.all-contributorsrc

```diff
@@ -32,6 +32,16 @@
       "contributions": [
         "code"
       ]
+    },
+    {
+      "login": "juniormartinxo",
```

Comment:
You don't need to change this file. This will be done automatically after the pull request is merged.